src/layouts/Home.tsx

```diff
@@ -1,183 +1,187 @@
-import React, { ChangeEvent, FC } from "react";
-import "./home.scss";
-import GoogleButton from "react-google-button";
-import { Link as MuiLink, TextField } from "@mui/material";
-import { RoundedButton } from "../elements/button";
-import { Link } from "react-router-dom";
-import { HomeContainer, HomeMain, InputForm, Left, Right } from "./Homestyles";
+import React, { ChangeEvent, FC } from "react"
+import "./home.scss"
+import GoogleButton from "react-google-button"
+import { Link as MuiLink, TextField } from "@mui/material"
+import { RoundedButton } from "../elements/button"
+import { useNavigate } from "react-router-dom"
+import { HomeContainer, HomeMain, InputForm, Left, Right } from "./Homestyles"
 
 export interface IHome {
-  title: string;
-  googleTitle: string;
-  buttonText: string;
-  navigation: {
-    title: string;
-    linkText: string;
-    link: string;
-  };
+	title: string
+	googleTitle: string
+	buttonText: string
+	navigation: {
+		title: string
+		linkText: string
+		link: string
+	}
+	requestFunction: () => void
 }
 
 interface ICredentials {
-  email: string;
-  password: string;
+	email: string
+	password: string
 }
 
 interface IValidation {
-  helperText: string;
-  error: boolean;
+	helperText: string
+	error: boolean
 }
 
-const Home: FC<IHome> = ({ title, googleTitle, buttonText, navigation }) => {
-  const [credentialsForm, setCredentialsForm] = React.useState<ICredentials>({
```

Comment:
Its better to import useState from React and simply use useState for defining states.